### PR TITLE
Add Powershell DSC troubleshooting section

### PR DESCRIPTION
**Pull request by octocrock**

### CODE REVIEW

File: docs/infrastructure/deployment-targets/windows-targets/azure-virtual-machines/via-an-arm-template-with-dsc.md [hunk 1]
```diff
@@ -480,3 +481,12 @@ az group deployment create \
     --template-file "arm-template.json" \
     --parameters "@arm-template.parameters.json"
 ```
+
+### Troubleshooting
```
Comment (harrisonmeister) [inline docs/infrastructure/deployment-targets/windows-targets/azure-virtual-machines/via-an-arm-template-with-dsc.md:485]:
```suggestion
## Troubleshooting 
```

File: docs/infrastructure/deployment-targets/windows-targets/azure-virtual-machines/via-an-arm-template-with-dsc.md [hunk 2]
```diff
@@ -480,3 +481,12 @@ az group deployment create \
     --template-file "arm-template.json" \
     --parameters "@arm-template.parameters.json"
 ```
+
+### Troubleshooting 
+
+To troubleshoot the installation, you can use [`Start-Transcript`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.host/start-transcript?view=powershell-7.1) to write the Powershell session to a file.
+
+In addition to this, if you're able to remote on the machine in question to find out the state of DSC, these two commands may offer diagnostic information:
```
Comment (harrisonmeister) [inline docs/infrastructure/deployment-targets/windows-targets/azure-virtual-machines/via-an-arm-template-with-dsc.md:489]:
```suggestion
In addition, if you have remote access to the machine you are troubleshooting installation for, these two commands may offer diagnostic information about the state of DSC:
```

**Review by harrisonmeister — COMMENTED** [review]
Looks good, made a couple of suggestions :+1:

**Review by twerthi — APPROVED** [review]
I agree with Mark's suggestions.

**Review by harrisonmeister — APPROVED** [review]

**Pull request merged**